content/docs/how-to-guides/import-an-existing-database.md

```diff
@@ -4,28 +4,68 @@ redirectFrom:
   - /docs/cloud/tutorials
 ---
 
-Importing an existing PostgreSQL database to Neon can be done in almost the same way
-as copying a database from one server to another. PostgreSQL provides several import methods.
-The most efficient and straightforward method is using the `pg_dump` utility.
+This topic describes how to import an existing PostgreSQL database to Neon. The instructions can also be used to migrate a database from one Neon project to another. For example, you can use the instructions to migrate a database from a Neon project created with PostgreSQL 14 to a Neon project created with PostgreSQL 15.
+
+PostgreSQL provides several import methods. The most efficient method is using the `pg_dump` utility with connection strings, as shown:
+
+```bash
+pg_dump <connection-string> | psql <connection-string>
+```
+
+The format for a PostgreSQL connection string is:
 
 ```bash
-pg_dump -h <host> -U <username> <dbname> | psql -h pg.neon.tech
+postgres://<user>:<password>@<hostname>:<port>/<dbname>
 ```
 
-In the `pg_dump` command shown above, replace `<host>`, `<user>`, and `<dbname>` with the appropriate values for the database you are importing.
+where:
+
+- `<user>` is your PostgreSQL user.
+- `<password>` is your PostgreSQL user's password.
+- `<hostname>` is the host name of the PostgreSQL instance.
+- `<port>` is the port number of the PostgreSQL instance. The default port number is `5432`.
+- `<dbname>` is the name of the database.
+
+A Neon connection string has this format:
```

Comment:
Maybe highlight the fact that Neon connection string is the same thing with `<hostname>` = `<project_id>.cloud.neon.tech`? But not sure here

Reply:
Revised.